usaspending_api/etl/management/commands/load_query_to_delta.py

```diff
@@ -0,0 +1,91 @@
+import uuid
+
+from django.core.management.base import BaseCommand
+from pyspark.sql import SparkSession
+from pyspark.sql.types import StringType
+
+from usaspending_api.common.helpers.spark_helpers import (
+    configure_spark_session,
+    get_active_spark_session,
+    get_jvm_logger,
+)
+from usaspending_api.search.models import TransactionSearch
+from usaspending_api.transactions.delta_models import (
+    transaction_search_create_sql_string,
+    transaction_search_load_sql_string,
+)
+
+TABLE_SPEC = {
+    "transaction_search": {
+        "model": TransactionSearch,
+        "source_query": transaction_search_load_sql_string,
+        "destination_database": "rpt",
+        "partition_column": "transaction_id",
+        "delta_table_create_sql": transaction_search_create_sql_string,
+        "custom_schema": "recipient_hash STRING, federal_accounts STRING",
+        "user_defined_functions": [
+            {
+                "name": "format_as_uuid",
+                "f": lambda val: str(uuid.UUID(val)) if val else None,
+                "returnType": StringType(),
+            },
+        ],
```

Comment:
There is a level of assumption with this function. It will handle the casing in regards to the output of MD5 / format of the UUID because that will always be lowercase. It does not handle the casing of values provided. It is assumed that the values reaching this are either upper case or lower case prior to reaching MD5 (our current approach for use of MD5 in Postgres).

Reply:
ok. A quick test though looks like if `val` is an uppercase UUID, this will normalize it to lowercase, which is good --> everything coming out of this function is lowercased.